### PR TITLE
Update Okta_v2.py

### PR DESCRIPTION
**Pull request by sorkan**
Added functions for listing users configured for a tenant on okta.

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Add additional command to okta V2 for getting list of users configured for a domain/tenant account

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [X ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Okta/Integrations/Okta_v2/Okta_v2.py [hunk 1]
```diff
@@ -849,6 +865,27 @@ def get_group_members_command(client, args):
...
+def list_users_command(client, args):
+    raw_response = client.list_users(args)
+    verbose = args.get('verbose')
+    users = client.get_readable_users(raw_response, verbose)
+    user_context = client.get_users_context(raw_response)
+    context = createContext(user_context, removeNull=True)
+    outputs = {
+        'Account(val.ID && val.ID == obj.ID)': context
+    }
+    if verbose == 'true':
```
Comment (DeanArbel) [inline Packs/Okta/Integrations/Okta_v2/Okta_v2.py:877]: I see this argument is not being used. Was this added only for testing?

**Review by DeanArbel — COMMENTED** [review]
Thank you very much for this contribution.
It's a well-written contribution, however, I have a small comment regarding one of the arguments you used - `verbose` - seems it was added only for testing. Is that correct?

The code looks good, but before we can continue, I'd like to ask can you please show me a short demo of this new command?
Please suggest a time that's convenient for you.

**Comment by DeanArbel** [comment]
@yaron-libman Please assign some from your team to review the release notes and the newly added command in the `yaml`. 

@sorkan After they're done with their technical writing review, we'll want to add the changes to the `README` as well. If you're having issues with that, I can assist.

**Comment by DeanArbel** [comment]
@sorkan Please make sure to sign the `Contributor Details`.
https://forms.gle/XDfxU4E61ZwEESSMA

**Comment by sorkan** [comment]
> @yaron-libman Please assign some from your team to review the release notes and the newly added command in the `yaml`.
> 
> @sorkan After they're done with their technical writing review, we'll want to add the changes to the `README` as well. If you're having issues with that, I can assist.


@DeanArbel :  How do I resolve the following validation errors?
=========== Found errors in the following files ===========

Packs/Okta/ReleaseNotes/2.2.10 - [BA102]
Packs/Okta/Integrations/Okta_v2/Okta_v2.yml - [DO106]
Packs/Okta/Integrations/Okta_v2/Okta_v2.yml - [BC104]
/home/circleci/project/Packs/Okta/pack_metadata.json - [RN106]

The files were found as invalid, the exact error message can be located above

**Comment by DeanArbel** [comment]
@sorkan Sure, to handle errors you should:
- RN106 + BA102 - change the release notes suffix to `.md`.
- DO106 - in the `Okta_v2.yml` the docker image has been updated in dockerhub, so it should be changed to `3.9.9.25564`.
- BC104 - seems you accidentally changed something in `okta-get-user-factors` command (in the yml file). Don't worry about this error, I'll fix it after the review is done.

**Comment by ShirleyDenkberg** [comment]
@sorkan @DeanArbel Doc review completed. Note that the Release Notes file should be renamed to 2_2_10.md

**Comment by sorkan** [comment]
> @sorkan @DeanArbel Doc review completed. Note that the Release Notes file should be renamed to 2_2_10.md

Still getting this error:
Packs/Okta/Integrations/Okta_v2/Okta_v2.yml: [BC104] - Possible backwards compatibility break, You've changed the name of a command or its arg in the file, please undo, the command was:
okta-get-user-factors

is there an order in which the new command details had to be added?

**Comment by DeanArbel** [comment]
@sorkan Thank you very much for your contribution!
I'll handle this from here. It should be ready by tomorrow.

**Comment by DeanArbel** [comment]
> @sorkan Please make sure to sign the `Contributor Details`. https://forms.gle/XDfxU4E61ZwEESSMA

@sorkan Please fill the form so we can proceed with emerging the pull request.

File: Packs/Okta/Integrations/Okta_v2/Okta_v2.yml [hunk 2]
```diff
@@ -93,6 +93,68 @@ script:
...
+    - contextPath: Account.Activated
+      description: Timestamp for when the user was activated.
+      type: Date
+    - contextPath: Account.StatusChanged
+      description: Timestamp for when the user's status was last changed.
+      type: Date
+    - contextPath: Account.PasswordChanged
+      description: Timestamp for when the user's password was last changed.
+      type: Date
+  - arguments:
```
Comment (DeanArbel) [inline Packs/Okta/Integrations/Okta_v2/Okta_v2.yml:127]:
```suggestion
  - description: Lists users in your organization.
    execution: false
    name: okta-list-users
    outputs:
    - contextPath: Account.ID
      description: Okta account ID.
      type: String
    - contextPath: Account.Email
      description: Okta account email.
      type: String
    - contextPath: Account.Username
      description: Okta account username.
      type: String
    - contextPath: Account.DisplayName
      description: Okta account display name.
      type: String
    - contextPath: Account.Status
      description: Okta account status.
      type: String
    - contextPath: Account.Created
      description: Timestamp for when the user was created.
      type: Date
    - contextPath: Account.Activated
      description: Timestamp for when the user was activated.
      type: Date
    - contextPath: Account.StatusChanged
      description: Timestamp for when the user's status was last changed.
      type: Date
    - contextPath: Account.PasswordChanged
      description: Timestamp for when the user's password was last changed.
      type: Date
    arguments:
```

**Comment by DeanArbel** [comment]
@sorkan I have fixed the error you were getting. The new command didn't have a `-` prefix in the right place, so the yaml treated the new output fields as if they were for the previous command in the yml file.

**Comment by DeanArbel** [comment]
@sorkan I tried fixing the issue, but I see you placed the new command in the middle of a previous command...
I'll need to edit it further, please hold on.

**Comment by sorkan** [comment]
> > @sorkan Please make sure to sign the `Contributor Details`. https://forms.gle/XDfxU4E61ZwEESSMA
> 
> @sorkan Please fill the form so we can proceed with merging the pull request.

@DeanArbel :I fixed the issue.  Everything built successfully on circleci.

**Review by DeanArbel — APPROVED** [review]

**Comment by DeanArbel** [comment]
@sorkan Good job. We'll merge these changes later today.
Thank you very much for your contribution!

**Pull request merged**